documentation/release_notes/release_notes_5.2.markdown

```diff
@@ -18,6 +18,15 @@ canonical: "/puppetdb/latest/release_notes.html"
 
 ---
 
+## PuppetDB 5.2.16
+
+PuppetDB 5.2.16 is a security release.
```

Comment:
```suggestion
Released X July 2020.
```

Reply:
Actually I see you don't usually add the release date like Puppet + Server do, is there a reason for this?

Reply:
I don't think there is a reason, probably just a historical accident. I'll update it to follow the Puppet/Server format.